### PR TITLE
Feature/trs2 documentation

### PR DESCRIPTION
**Pull request by johan-riscure**
Please make sure to finish the review and merge of the trace parameter feature branch first.
If that is done and this branch is ok too, this be a complete version and ready for a merge to master, which is why I'm making this pull request there.

### CODE REVIEW

**Review by Siebje — COMMENTED** [review]
Readme looks fine, but this also contains all the other changes, so those need to be merged first I guess. I'm not setting it to approved yet because of that.

File: README.md [hunk 1]
```diff
@@ -32,18 +106,23 @@ with trsfile.open('trace-set.trs', 'r') as traces:
...
 
 with trs_open(
 		'trace-set.trs',                 # File name of the trace set
 		'w',                             # Mode: r, w, x, a (default to x)
 		# Zero or more options can be passed (supported options depend on the storage engine)
 		engine = 'TrsEngine',            # Optional: how the trace set is stored (defaults to TrsEngine)
 		headers = {                      # Optional: headers (see Header class)
-			Header.LABEL_X: 'Testing X',
-			Header.LABEL_Y: 'Testing Y',
+			Header.TRS_VERSION: 2,
```
Comment (matthijsgeers-riscure) [inline README.md:120]: Should creation of a TRSv1 trace set still be allowed? The Java library doesn't allow this. I'd say this header should be implicitly set to version 2 for every new trace set created

Comment (johan-riscure) [reply README.md:120]: The version is already set to 2 by default if you don't specify it, or if you use the new trace parameter features. So even though you can still explicitly set it to 1, I don't think that's an issue

Comment (matthijsgeers-riscure) [reply README.md:120]: You're right; I missed it, but it's there after all. In that case, explicitly setting it should probably be removed from this code example, or users might think it's required

**Review by Siebje — APPROVED** [review]

**Pull request merged**